src/ui-lib/sass/00-settings/typography.scss

```diff
@@ -1,7 +1,10 @@
 /**
   Typography-related values
 */
+@import url('https://fonts.googleapis.com/css?family=Cabin:400:600|Roboto');
 
 $font-size-base: unset;
-$line-height-default: 1.2;
+// TODO remove and use gravy `$base-font-size` and `$base-line-height`
```

Comment:
The existing variables were just placeholders. Please just go ahead and remove them and reference the Gravy-generated ones directly where needed.